Contributing.md

```diff
@@ -0,0 +1,56 @@
+# Contributing
+
+Everyone is welcome to contribute to facebook-containers. Reach out to team members if you have questions:
+
+- IRC: #containers on irc.mozilla.org
+- Email: containers@mozilla.com
+
+## Filing bugs
+
+If you find a bug with facebook-container, please file a issue.
+
+Check first if the bug might already exist: https://github.com/mozilla/contain-facebook/issues
+
+[Open an issue](https://github.com/mozilla/contain-facebook/issues/new)
+
+1. Visit about:support
+2. Click "Copy raw data to clipboard" and paste into the bug. Alternatively copy the following sections into the issue:
+  - Application Basics
+  - Nightly Features (if you are in nightly)
+  - Extensions
+  - Experimental Features
+3. Include clear steps to reproduce the issue you have experienced.
+4. Include screenshots if possible.
+
+## Installing
```

Comment:
Remove this section. I would like to add documentation for setting the project up to the README itself. (Similar to how MAC documents it.)